src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -36,10 +36,17 @@ This package is an internal implementation of the .NET Core SDK and is not meant
 
     <!-- Reference implementation assemblies in addition to ref assemblies to get xml docs -->
     <ReferenceImplementationAssemblies>true</ReferenceImplementationAssemblies>
+    <!-- We are ignoring MSB3243 warnings since implemenation and reference assemblies are versioned differently. We need both to compose the targeting pack with reference assemblies and xml docs. -->
+    <MSBuildWarningsAsMessages>MSB3243</MSBuildWarningsAsMessages >
```

Comment:
Remove this workaround if possible. Should not be necessary since everything has the same assembly versions now.

Reply:
I think we'll want to eventually take the approach of building against ref assemblies which means this will come back. Also, I don't want to respin the builds again. But I'll definitely file a follow up issue for further work and mention it there.